src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const rows = [...table.rows];
+
+rows.forEach((row) => {
+  const copy = row.cells[1].cloneNode(true);
+
+  row.cells[4].before(copy);
+});
```

Comment:
This code assumes that each row has at least five cells. If a row has fewer than five cells, `row.cells[4]` will be undefined, leading to an error when trying to use the `before` method. Consider adding a check to ensure that the row has enough cells before attempting to clone and insert.